doc/guide/guide-basics.rst

```diff
@@ -11,21 +11,21 @@ First things first
 
 .. warning:: Do not run QuTiP from the installation directory.
 
-To load the qutip modules, first call the import statement:
+To load the qutip modules, we must first call the import statement:
 
 .. code-block:: Python
 
    from qutip import *
 
-This will load all of the user available functions. Often, we also need to import the NumPy and Matplotlib libraries with:
+that will load all of the user available functions. Often, we also need to import the NumPy and Matplotlib libraries with:
 
 .. code-block:: Python
 
    import numpy as np
 
    import matplotlib.pyplot as plt
 
-In the rest of the documentation, functions are written using `qutip.module.function()` notation which links to the corresponding function in the QuTiP API: :ref:`functions`. However, in calling `import *`, we have already loaded all of the QuTiP modules. Therefore, we will only need the function name and not the complete path when calling the function from the interpreter prompt, Python script, or Jupyter notebook.
+Note that, in the rest of the documentation, functions are written using `qutip.module.function()` notation which links to the corresponding function in the QuTiP API: :ref:`functions`. However, in calling `import *`, we have already loaded all of the QuTiP modules. Therefore, we will only need the function name and not the complete path when calling the function from the interpreter prompt, Python script, or Jupyter notebook.
```

Comment:
I don't know if the `Note that` here is really improving things -- it's just adding more words to little effect.